### PR TITLE
Modularize Application Centric Deployment Guide

### PR DESCRIPTION
**Pull request by maximiliankolb**
This PR modularizes the ACD guide further to ensure each file has only one heading.
Sadly, the git diff here on Github looks massive due to renaming files.

Cherry-pick into:

* [x] Foreman 3.1

### CODE REVIEW

**Comment by melcorr** [comment]
@maximiliankolb I can review this on Friday and all your other PRs. 
Hope that's OK.

**Comment by maximiliankolb** [comment]
> @maximiliankolb I can review this on Friday and all your other PRs. Hope that's OK.

That would be great; Friday sounds good!

File: guides/common/modules/con_ansible-variables-and-parameter-hierarchy.adoc [hunk 1]
```diff
@@ -0,0 +1,29 @@
+[id="Ansible_Variables_and_Parameter_Hierarchy_{context}"]
+= Ansible Variables and Parameter Hierarchy
+
+There are different levels on how to set Ansible variables and parameters for a service/host.
```
Comment (melcorr) [inline guides/common/modules/con_ansible-variables-and-parameter-hierarchy.adoc:4]: This is a nitpick, off-topic, and a non blocker but something that crushes my technical writer heart. I've literally no idea what we're talking about. Can we stop equating service and host and just say host or service and host or service or host if they are different thing? I feel this is a weak spot.

**Review by melcorr — CHANGES_REQUESTED** [review]
I built and read through. 
The only thing that I would say is a cautionary note on the capitalisation. 

1. Playbook is not capitalised in Ansible playbook, as per Ansible's own documentation. 
2. "Application Definitions" capitalised is bordering on problematic but I will not block it. It wouldn't be a goer in Satellite docs. 

There's a capitalisation section in IBM style that is a good guide. 
The German language, from what I know of it, capitalises common nouns, while other languages don't. Feeding capitalised terms through a machine translator can lead to absurdities for the reader. I read a really good book about that when I was younger. 
I find that engineers tend to capitalise all of the new things that they develop, treating new features like a newborn child that deserves a Capital letter. IBM style says otherwise. 
This is your domain, and not a blocker, past the playbooks. That is just wrong. 

Other than that, everything is perfect as far as my eyes can tell. 

Thank you so much for your work and for your talk yesterday.

File: guides/common/modules/con_ansible-variables-and-parameter-hierarchy.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/common/modules/con_ansible-variables-and-parameter-hierarchy.adoc:4]: Removed `service/`.

**Comment by maximiliankolb** [comment]
Rebased to master and applied one suggestion.

Regarding capitalization: right now, we use title case almost everywhere; even though IBM Style Guide recommends using sentence case. If we change this, we should do that in an automated way. I cannot spot any error apart from that:

```
$ git grep "Ansible Playbook"
guides/common/modules/con_ansible-playbooks.adoc:= Ansible Playbooks
guides/common/modules/con_ansible-playbooks.adoc:In the {ProjectWebUI}, navigate to *Applications > Ansible Playbooks* to manage Ansible playbooks for application centric deployment.
guides/common/modules/con_key-terms-in-application-centric-deployment.adoc:Ansible Playbooks::
guides/common/modules/con_user-roles-for-application-centric-deployment.adoc:* Ansible playbooks: *Applications > Ansible Playbooks*
guides/common/modules/proc_adding-an-ansible-playbook.adoc:= Adding an Ansible Playbook
guides/common/modules/proc_adding-an-ansible-playbook.adoc:. In the {ProjectWebUI}, navigate to *Applications > Ansible Playbooks*.
guides/common/modules/proc_adding-an-ansible-playbook.adoc:. Click *New Ansible Playbook*.
guides/common/modules/proc_creating-an-application-definition.adoc:. From the *Ansible Playbook* list, select the name of an Ansible playbook.
guides/common/modules/proc_customizing-an-ansible-playbook-to-reconfigure-hosts.adoc:= Customizing an Ansible Playbook to Reconfigure Hosts
guides/common/modules/proc_removing-an-ansible-playbook.adoc:= Removing an Ansible Playbook
guides/common/modules/proc_removing-an-ansible-playbook.adoc:. In the {ProjectWebUI}, navigate to *Applications > Ansible Playbooks*.
guides/common/modules/proc_running-the-ansible-playbook.adoc:= Running the Ansible Playbook
guides/common/modules/proc_viewing-an-ansible-playbook.adoc:= Viewing an Ansible Playbook
guides/common/modules/proc_viewing-an-ansible-playbook.adoc:. In the {ProjectWebUI}, navigate to *Applications > Ansible Playbooks*.
```

Thanks for the review!

**Comment by melcorr** [comment]
Title case pertains only to titles. 
You must use chapter 7 Computer interfaces for guidance on GUI labels. 
The advice is that the docs must match what is in the web UI. IBM Style pp 207. 
Sorry :(

**Comment by melcorr** [comment]
Cherry pick: 
   a48a6193..aef7e404  3.1 -> 3.1

**Pull request merged**